packages/front-end/components/ProtectedPage.tsx

```diff
@@ -56,11 +57,13 @@ export type UserContextValue = {
   refreshUsers?: () => Promise<void>;
   permissions: Permissions;
   settings: OrganizationSettings;
+  connections?: OrganizationConnections;
```

Comment:
Mentioned above, but don't send connections to the front-end since they contain secret tokens.